utilities/sourceEvents.py

```diff
@@ -150,6 +150,9 @@ def parse(content, gmaps):
         entry['sourceId'] = '0'
         entry['allDay'] = False
 
+        if 'costFree' in pe:
```

Comment:
@bingzhang. This needs to be converted to boolean type like here: https://github.com/rokwire/events-manager/pull/646/files#diff-4e3a3415696b4b1b4c587a65633d38875ede9be2a8c48ad41013c6ccb47eeeacR136.